src/test/regress/sql/aqumv.sql

```diff
@@ -795,6 +795,41 @@ select count(*), sum(c1) from t where c1 > 90 limit all;
 
 abort;
 
+--
+-- test partitioned tables
+--
+create table par(a int, b int, c int) partition by range(b)
+    subpartition by range(c) subpartition template (start (1) end (3) every (1))
+    (start(1) end(3) every(1));
+insert into par values(1, 1, 1), (1, 1, 2), (2, 2, 1), (2, 2, 2);
+insert into par values(1, 1, 1), (1, 1, 2), (2, 2, 1), (2, 2, 2);
+insert into par values(1, 1, 1), (1, 1, 2), (2, 2, 1), (2, 2, 2);
+create materialized view mv_par as select    count(*) from par;
+create materialized view mv_par1 as select   count(*) from par_1_prt_1;
+create materialized view mv_par1_1 as select count(*) from par_1_prt_1_2_prt_1;
+create materialized view mv_par1_2 as select count(*) from par_1_prt_1_2_prt_2;
+create materialized view mv_par2 as select   count(*) from par_1_prt_2;
+create materialized view mv_par2_2 as select count(*) from par_1_prt_2_2_prt_1;
+create materialized view mv_par_prune as select count(*) from par where b = 1;
+set enable_answer_query_using_materialized_views = on;
```

Comment:
What about the pg partition to support AQUMV? If a new partition is attached/detached, is it possible to return wrong result?

Reply:
Thanks for review.
 These cases are handled in ATTACH/DETACH command, in general, attach to a table p is equal to insert into p , and detach form p is equal to  delete from p. There are test cases about those in matview_data.sql